packages/botonic-plugin-flow-builder/src/functions/conditional-queue-status.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+export async function conditionalQueueStatus({ queue_id }): Promise<string> {
+  const response = await axios.get(
+    `https://api.hubtype.com/v1/queues/${queue_id}/availability/`
+  )
+  const isAvailable = response.data.available
+  return isAvailable ? 'open' : 'closed'
+}
```

Comment:
Just realized status does not depend in just open or closed queue. It also depends on agent available per queue and queue threshold.
https://github.com/metis-ai/hubtype-backend/blob/develop/organization/public_serializers/queue.py#L17

Just wanted to point it out, maybe a queue is not available at the moment of the assignation, but it's open. We should talk about this with @ericmarcos and product about what behaviour do we want.

Reply:
yep, added it 👍